R/task_bulk.R

```diff
@@ -50,6 +52,13 @@ bulk_task_save <- function(X, FUN, context, DOTS = NULL,
   }
 
   n <- length(dat)
+  if (!is.null(depends_on)) {
+    nd <- length(depends_on)
+    if (nd < n) {
+      stop(sprintf(paste("Failed to save as 'depends_on' must be of",
```

Comment:
do you want to support the shorthand of `depends_on` being a single vector shared across everything? so if `depends_on` is a character vector of ids then you'd do `depends_on <- rep(list(depends_on), n)`?

Reply:
it's a different type though in that case (a list vs a character vector), which should make it less ambiguous